Packs/CommonScriptsV2/Scripts/GetListDatawithKeyword/GetListDatawithKeyword.yml

```diff
@@ -0,0 +1,41 @@
+args:
```

Comment:
what about the json_data?
Also, let's give it a better name than 'value', let's call it **json_list**.

Reply:
Not required i think so, the list of json_objects being stored in the argument value